pom.xml

```diff
@@ -19,7 +19,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.jongo</groupId>
+    <groupId>com.evergage.thirdparty.jongo</groupId>
```

Comment:
Can we just make the groupId `com.evergage.thirdparty`?

Reply:
I went with the pattern `com.evergage.thirdparty.dependencyname` because there's already a `com.evergage.thirdparty.elasticsearch`. It seemed the consistent thing to do.
I don't have a strong opinion one way or the other, tbh. @rmichela might have one.

Reply:
@ispringer That would make the maven coordinate `com.evergage.thirdparty:1.5.0-evg5`, which loses the original name of the dependency. `com.evergage.thirdparty.dependencyname` is more descriptive.

Reply:
Iff we opt to put all third party dependencies under `com.evergage.thirdparty`, for consistency's sake we should relocate the `es-shaded` repo pom there from its pre-existing home at `com.evergage.thirdparty.elasticsearch`.